doc/nrf/libraries/others/nrf_profiler.rst

```diff
@@ -1,4 +1,4 @@
-.. _profiler:
+.. _nrf_profiler:
 
 Profiler
```

Comment:
Let's maybe follow the naming change and call this `nRF Profiler`? Could you check with @pdunaj and PMTs (if PMTs need to be involved in this)?